server.js

```diff
@@ -54,6 +56,8 @@ function parseArguments() {
                 throw new Error("Live reload is only supported in development mode");
             }
             liveReload = true;
+        } else if (args[i] === "--skip-data-update") {
+            skipDataUpdate = true;
```

Comment:
I needed to skip the data update on each startup as I was restarting a lot and didn't want to hammer the store APIs.